### ISSUE TITLE
Finding data for AlexdeBruyn

### ISSUE THREAD

**Issue by aammd** [BIOL548O/Discussion#9]
Hello @AlexdeBruyn,

In the pre-course survey, you indicated that you don't have any data. Experience with real data is more useful to you than working through exercises with your own messy data. Therefore, I wanted to have a discussion with you about what kind of data you might be able to use.  

Here are some suggestions (ranked _very_ loosely from most to least helpful):  
- [ ] Data from a pilot experiment  
- [ ] Data from your supervisor / collaborator / senior grad student   (assure them that these data will remain confidential and unpublished)
- [x] Data from any previous degree you've done (Honours / Masters). This would be especially useful to you if the data are unpublished (even if the paper _is_ published)
- [ ] Published data from a paper in your field
- [ ] Data emailed to you from an author in your field
- [ ] Data (relevant to your work) obtained from the Web -- ie from a data archive or elsewhere. If you know where to look, i can help you get it.
## Plan B

If none of the above datasets work, we'll have to find something else. This could be a part of the Kaggle dataset, the candy dataset, or some other random dataset you enjoy. I could also give you some of my own data which are still unpublished.

Which of these options appeals most to you? Let's try to figure this out by Thursday.  

Andrew

**Comment by AlexdeBruyn** [comment]
I've got a bunch of data showing relative abundances of different orders of insect at different experimental sites that I collected for my undergraduate thesis. Could that potentially work?

Date: Mon, 8 Feb 2016 22:59:05 -0800

**Comment by aammd** [comment]
> relative abundances of different orders of insect at different experimental sites

I mean, relative abundances of insects are my **favourite** so yes, I support this :100: %
That said, do you have the original, raw, count data? That would be even more fun than the relative abundances (by which I assume you mean the total abundance of each species, divided by the total abundance of all species)

**Comment by AlexdeBruyn** [comment]
Ironically enough for the theme of this course, I don't think I have the digitized data on my curent computer. I'll look through my old emails to see if I can find it in any attachments. However, I can get my parents back home to look at my lab notebook and re-send me the data set.

Date: Tue, 9 Feb 2016 05:37:11 -0800

**Comment by aammd** [comment]
Sounds good.  Ideally you'd have it by Thursday, which is when we're going to start work on our individual projects. If that seems unworkable, do you have a backup option? I'm sure I could find you some insect count data somewhere, though it might be pretty gnarly :stuck_out_tongue_winking_eye:

**Comment by aammd** [comment]
Hey Alex. This choice of data sounds great for the course. The table of different insect orders that you showed me earlier looks to me like a summary of the original data. i imagine that you collected these insects in a series of samples. Do you have that raw data, showing which insects are found in which sample?

**Comment by aammd** [comment]
@AlexdeBruyn any luck finding those original data, at the sample level? Just curious :smile_cat:

**Comment by AlexdeBruyn** [comment]
Each site is actually a single giant sample, collected over a 24 hour period. So yes, one single sample gathered over 2000 flies into a small bottle of alcohol, hahah. The dataset that I have provided contains all of the original information, just converted from tally marks to numbers. The bit of data I am trying to add in is the subset of Ichneumonid Wasps collected from Tagish II, which I identified to the family level.
